bitnami/discourse/templates/_helpers.tpl

```diff
@@ -120,7 +120,29 @@ imagePullSecrets:
 {{- end -}}
 
 {{/*
-Get the user defined LoadBalancerIP for this release.
+Return the Discourse secret name
+*/}}
+{{- define "discourse.discourseSecretName" -}}
+{{- if not (empty .Values.discourse.existingSecret) }}
+    {{- .Values.discourse.existingSecret }}
```

Comment:
We can use the `printf` function here just to ensure proper indentation is followed.
```go
{{- printf "%s" .Values.discourse.existingSecret -}}
```